docs/website/data/integrations.yaml

```diff
@@ -744,6 +744,22 @@ integrations:
     code:
       - https://github.com/GluuFederation/gluu-gateway
 
+  pre-commit-hooks:
+    title: Pre-commit hooks
+    description: Pre-commit git hooks for OPA and Rego development
+    labels:
+      category: tooling
+      layer: cicd
+    software:
+      - git
+      - pre-commit
+    code:
+      - https://github.com/anderseknert/pre-commit-opa
+    blogs:
+      - https://www.eknert.com/tech/2020/08/31/pre-commit-hooks-for-opa
+    inventors:
+      - name: Anders Eknert
```

Comment:
From looking at the docs for the integration index stuff, I think we need an `organization` field here. That field has to link to an existing key in the `organizations` document, I believe. Could you add an `independent` organization under the `organizations` document and then put `organization: independent` here? I hope that fixes it.

Reply:
Link to build failure: https://app.netlify.com/sites/openpolicyagent/deploys/5f4e51c297b90b0007fd2903

Reply:
Sure thing. This has been a side project of mine so I didn't want to involve my employer here. I'll add `independent` as an organization and will assign this to that.